Code/GraphMol/Wrap/rough_test.py

```diff
@@ -3198,6 +3198,20 @@ def test84PDBBasics(self):
     self.assertEqual(atom.GetPDBResidueInfo().GetResidueName(), 'HOH')
     self.assertEqual(atom.GetDegree(), 0)
 
+  def test85AtomCopying(self):
+    """Can a copied atom be added to a molecule?"""
+    import copy
+    m = Chem.MolFromSmiles('C1CC1')
+    a = m.GetAtomWithIdx(0)
+    a_copy = copy.copy(a)
+    m = None
+    a = None
+    new_m = Chem.EditableMol()
```

Comment:
```suggestion
    new_m = Chem.RWMol()
```
would be better here.

Reply:
I'll add the __copy__ dunder method for bonds.

There are two reasons that I think that copying using a constructor is slightly less idiomatic in Python: 1. duck typing - using a ctor requires you to know the type of the object that you are copying. 2. Because Python classes don't can't overload constructors.

That being said, this is wrapped C++, so it makes sense to be able to mirror the copy calls in the two languages.